spec/features/person_creation_spec.rb

```diff
@@ -26,4 +26,20 @@
     expect(current_path).to eq everybody_people_path
   end
 
+  scenario "person can not be created if start date is before application date" do
```

Comment:
I love that you included a test, but this does not actually test what's in the scenario. This is because you only test the "happy" path. You should also have scenarios for start_date missing, application_date missing, and then they're being out of order.
Thanks for your help !